### PR TITLE
Adding dockerfile for Konflux

### PR DESCRIPTION
**Pull request by Vincent056**
We need to add the OpenShift docker file for Konflux.

### CODE REVIEW

File: images/el8/Dockerfile.openshift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        io.k8s.description="An operand for the Security Profiles Operator" \
+        io.openshift.tags="selinux,security,deamon" \
+        com.redhat.delivery.appregistry="false" \
+        maintainer="Red Hat ISC <isc-team@redhat.com>" \
+        License="APL 2.0" \
+        name="openshift-selinuxd" \
+        com.redhat.component="openshift-selinuxd-container" \
+        io.openshift.maintainer.product="OpenShift Container Platform" \
+        io.openshift.maintainer.component="Security Profiles Operator" \
+        version=0.8.4
```
Comment (rhmdnd) [inline images/el8/Dockerfile.openshift:39]: Do we need to update the version here to a dev version of some kind? I though I hit issues with the version being the same as a previous release when building the OLM dependency graph.

File: images/el9/Dockerfile.openshift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        io.k8s.description="An operand for the Security Profiles Operator" \
+        io.openshift.tags="selinux,security,deamon" \
+        com.redhat.delivery.appregistry="false" \
+        maintainer="Red Hat ISC <isc-team@redhat.com>" \
+        License="APL 2.0" \
+        name="openshift-selinuxd" \
+        com.redhat.component="openshift-selinuxd-rhel9-container" \
+        io.openshift.maintainer.product="OpenShift Container Platform" \
+        io.openshift.maintainer.component="Security Profiles Operator" \
+        version=0.8.4
```
Comment (rhmdnd) [inline images/el9/Dockerfile.openshift:45]: Similar comment here as above.

File: images/el8/Dockerfile.openshift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+RUN SEMODULE_BACKEND=policycoreutils make
+
+FROM registry.redhat.io/ubi8/ubi-minimal:latest
+
+USER root
+
+RUN INSTALL_PKGS="policycoreutils" && \
+    if [ ! -e /usr/bin/dnf ]; then ln -s /usr/bin/microdnf /usr/bin/dnf; fi && \
+        dnf install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
```
Comment (yuumasato) [inline images/el8/Dockerfile.openshift:23]: I would already add `dnf update -y`

File: images/el9/Dockerfile.openshift [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+RUN SEMODULE_BACKEND=policycoreutils make
+
+FROM registry.redhat.io/ubi9/ubi-minimal:latest
+
+USER root
+
+RUN INSTALL_PKGS="policycoreutils" && \
+    if [ ! -e /usr/bin/dnf ]; then ln -s /usr/bin/microdnf /usr/bin/dnf; fi && \
+        dnf install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
```
Comment (yuumasato) [inline images/el9/Dockerfile.openshift:29]: Same here.

**Pull request merged**